src/algebra/homology/Module.lean

```diff
@@ -80,14 +80,19 @@ example (f g : C ⟶ D) (h : homotopy f g) (i : ι) :
 begin
   -- To check that two morphisms out of a homology group agree, it suffices to check on cycles:
   ext,
-  dsimp,
-  simp only [homology.π_map_apply],
+  simp only [homology_functor_map, homology.π_map_apply],
   -- To check that two elements are equal mod boundaries, it suffices to exhibit a boundary:
-  ext1,
-  swap, exact (to_prev i h.hom) x.1,
+  ext1, swap,
+  exact (to_prev i h.hom) x.1,
```

Comment:
Is there a reason that you switched the place of the newline? I like it more when the `swap` is placed together with the tactic that discharges the `swap`ped goal.
```suggestion
  ext1,
  swap, exact (to_prev i h.hom) x.1,
```

Reply:
No, I just had to mess around with the proof before figuring out how to speed it up and this change remained.